### PR TITLE
Possible edits.

### PR DESCRIPTION
**Pull request by arfon**
Is it possible to use even stronger language?

### CODE REVIEW

File: Repositories/Readme.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
...
 
-  AAS Journals encourage authors to place data or software products related to an article into a persistent repository linked to the main journal article by a DOI and listed in the main references. See subsequent sections in this tutorial on formatting those references, and inserting them into your manuscript.
+  AAS Journals strongly encourage authors to place data or software products related to an article into a persistent repository linked to the main journal article by a DOI and listed in the main references. See subsequent sections in this tutorial on formatting those references, and inserting them into your manuscript.
```
Comment (arfon) [inline Repositories/Readme.md:10]: Requires?

Comment (augustfly) [inline Repositories/Readme.md:10]: I wish, but that produces a cascade of policy decisions...

Comment (arfon) [inline Repositories/Readme.md:10]: Strongly then?

Comment (augustfly) [inline Repositories/Readme.md:10]: It might become policy if/when we have our own repository (or a contractual relationship with one).

Comment (arfon) [inline Repositories/Readme.md:10]: Fair enough.

**Comment by arfon** [comment]
:+1:

**Pull request merged**